contracts/src/rollup/RollupCore.sol

```diff
@@ -409,6 +409,13 @@ abstract contract RollupCore is IRollupCore, PausableUpgradeable {
             require(afterInboxPosition >= prevInboxPosition, "INBOX_BACKWARDS");
```

Comment:
Can't this be `>` instead of  `>=` ? (since logic below ensures assertion can't have 0 messages)?

Reply:
It cant, because of the error case

Reply:
ah i see, ty